cli.py

```
@@ -1,67 +1,90 @@
+        config = yaml.safe_load(f)
+
+    HORIZONS = config["features"]["horizons"]
+    PREDICTORS = [f"{h}_day" for h in HORIZONS]
+
+    if not os.path.exists(MODEL_PATH):
+        print(f"Error: Model file not found at '{MODEL_PATH}'.")
+        print("Please run the training pipeline first: `python train.py`")
+        return
+    try:
+        model = joblib.load(MODEL_PATH)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)